testgrid/cmd/transfigure/BUILD.bazel

```diff
@@ -0,0 +1,28 @@
+load("@io_bazel_rules_docker//container:image.bzl", "container_image")
+
+container_image(
+    name = "image",
+    base = "@gcloud-go//image",
+    directory = "/",
+    entrypoint = ["/transfigure.sh"],
+    files = [
+        "transfigure.sh",
+        "//robots/pr-creator",
```

Comment:
This looks like a better pattern than making the image download and compile pr-creator with the `go run` command. Can we a) switch the script to do this and b) do this for configurator as well?